### PR TITLE
Feature/cdn disable

### LINKED ISSUE #767: Add CDN_missing_configuration query for AWS CloudFormation
### Platform
AWS

### Provider
AWS

### Description
Content Delivery Network (CDN) service is used within AWS account to secure and accelerate the delivery of websites. The use of a CDN can provide a layer of security between your origin content and the destination.

### PR DESCRIPTION
**Pull request by soniantao**
Closes #768
Closes #767

### CODE REVIEW

File: assets/queries/cloudFormation/disable_CDN/metadata.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "id": "CDN_missing_configuration",
+  "queryName": "CDN_missing_configuration",
```
Comment (joaomartinscx) [inline assets/queries/cloudFormation/disable_CDN/metadata.json:3]: queryName should have a verb to easily identify  the resource and the wrong action, in this case, something like: "CDN Configuration Is Missing"

File: assets/queries/cloudFormation/disable_CDN/query.rego [hunk 2]
```diff
@@ -0,0 +1,17 @@
+package Cx
+
+CxPolicy [result] {
+  resource := input.document[i].Resources[name]
+  properties := input.document[i].Resources[name].Properties[j]
+  distributionConfig := properties.Enabled
```
Comment (joaomartinscx) [inline assets/queries/cloudFormation/disable_CDN/query.rego:6]: Should check the resource type, if it "AWS::CloudFront::Distribution"

File: assets/queries/cloudFormation/disable_CDN/query.rego [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  expectedvalue := "false"
+  distributionConfig == expectedvalue
+  object.get(properties, "Origins", "undefined") == "undefined"
```
Comment (joaomartinscx) [inline assets/queries/cloudFormation/disable_CDN/query.rego:9]: What if only one of these checks is true, shouldn't also be caught?

**Review by joaomartinscx — CHANGES_REQUESTED** [review]

File: assets/queries/cloudFormation/disable_CDN/query.rego (hunk 3 above)
Comment (joaomartinscx) [reply assets/queries/cloudFormation/disable_CDN/query.rego:10]: This still hasn't been addressed. The check finds a failure when Not (Enabled Is True and Origins Is Defined), meaning, Enabled Is Not True or Origins Is Not Defined

**Review by joaomartinscx — CHANGES_REQUESTED** [review]

**Review by joaomartinscx — COMMENTED** [review]
Everything looks good to me, I recommend the approval and merge of this PR

**Review by ruigomescx — APPROVED** [review]

**Pull request merged**